deploy/olm-catalog/cluster-network-addons/0.0.0/cluster-network-addons-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -169,9 +169,6 @@ spec:
                       fieldRef:
                         fieldPath: metadata.name
                   - name: WATCH_NAMESPACE
-                    valueFrom:
```

Comment:
there is no value here it's fine?

Reply:
AFAIK it is. Value is of type `string` and if no value is provided it defaults to `""`.